lms/djangoapps/instructor/tasks.py

```diff
@@ -75,7 +77,10 @@ def make_subqueries(course_id, group_id, queries):
             query.filter_on,
             query.entity_name,
         )
-        make_single_query.apply_async(args=(course_id, query, group_id), kwargs={'origin': QueryOrigin.EMAIL})
+        tasks.append(make_single_query.si(course_id, query, group_id,origin=QueryOrigin.EMAIL))
+    res = group(tasks)()
+    # blocks until all single queries are finished
+    res.get()
```

Comment:
I think they will just get dropped. I think that is better than retrying because then we get into a bad loop